.github/pull_request_template.md

```diff
@@ -6,7 +6,9 @@
 * [ ] Resolved all conflicts, if any
 * [ ] Rebased your branch PR on top of the latest upstream `main` branch
 * [ ] Attached at least one of the following labels to the PR: `[chore, ci, docs, feat, fix, test]`
-* [ ] Signed the last commit: `git commit --amend --signoff`
+* [ ] Signed the last commit using a GPG signature: `git commit --amend --gpg-sign`
```

Comment:
I think it should actually instruct the PR author that *all* commits should be signed, not just the last one. Maybe it's worth adding links here to documentation on how to add a GPG key to the local gitconfig and the user's GitHub account?

Reply:
yes I can do that